android/build.gradle

```diff
@@ -29,20 +29,21 @@ buildscript {
     // This avoids unnecessary downloads and potential conflicts when the library is included as a
     // module dependency in an application project.
     // ref: https://docs.gradle.org/current/userguide/tutorial_using_tasks.html#sec:build_script_external_dependencies
-    if (project == rootProject) {
+    
+    // Condition removed to prevent the need to add kotlin plugin in integrating apps:
+    // if (project == rootProject) { 
```

Comment:
Given we have version control, could we just delete this instead of commenting out? Is this logic necessary in other scenarios where quickly removing the comments is helpful?

Reply:
comment was removed